tests/src/testVariableMapper.cpp

```diff
@@ -79,7 +79,7 @@ BOOST_AUTO_TEST_CASE(testEvaluateBool) {
     VariableMapper::evaluateBool("fale");
     BOOST_ERROR("testEvaluateBool did not throw as expected"); // LCOV_EXCL_LINE
   }
-  catch(std::logic_error& e) {
+  catch(std::exception& e) {
```

Comment:
why std::exception, not ChimeraTK::logic_error?

Reply:
std::exception also catches 'throw std::invalid_argument' used several times in VariableMapper.cc